aten/src/ATen/native/TensorConversions.cpp

```diff
@@ -1756,7 +1786,7 @@ Tensor sparse_compressed_to_sparse(const Tensor& self, c10::optional<c10::Layout
   }
   switch (layout_) {
   case kStrided:
-    return sparse_compressed_to_dense(self);
+    return sparse_compressed_to_dense(self, /*dtype=*/c10::nullopt, /*masked_grad=*/c10::nullopt);
```

Comment:
Wouldn't it be more clear to pass explicit `true/false` instead? All while leaving `nullopt` for something to indicate that the user (not the dev) did not provide any value.

Reply:
No. I am using `nullopt` to ensure the default behavior. In addition, `sparse_compressed_to_dense` does not use `masked_grad` value anyway.

Hmm, it is not clear to me how the specification of `dtype` was not required before...

Reply:
Actually, the right thing to do here would be to add `masked_grad` kw argument to `to_sparse` method as well so that the correct value could be propagated to `to_dense` call when the specified layout is `torch.strided`. However, changing `to_sparse` API is OT here and I suggest we leave the code as it is for `masked_grad` visibility here.